util.go

```diff
@@ -118,23 +119,41 @@ func toNamedValues(values []driver.Value) []driver.NamedValue {
 type TokenAccessor interface {
 	GetTokens() (token string, masterToken string, sessionID int)
 	SetTokens(token string, masterToken string, sessionID int)
+	Lock() error
+	Unlock() error
```

Comment:
We should either handle the error when unlocking in renewExpiredSessionToken() or remove the error return type.

Reply:
prefer to remove the error return type in the unlock, will remove